examples/mnist/estimator/mnist_spark.py

```diff
@@ -150,6 +150,6 @@ def parse(ln):
 
   images_labels = sc.textFile(args.images_labels).map(parse)
 
-  cluster = TFCluster.run(sc, main_fun, args, args.cluster_size, num_ps=0, tensorboard=args.tensorboard, input_mode=TFCluster.InputMode.SPARK, log_dir=args.model_dir, master_node='chief')
+  cluster = TFCluster.run(sc, main_fun, args, args.cluster_size, num_ps=0, tensorboard=args.tensorboard, input_mode=TFCluster.InputMode.SPARK, log_dir=args.model_dir, main_node='chief')
```

Comment:
This will unfortunately break backwards compatibility. Also, perhaps consider using some other word than `chief`.